static-analysis/autodispose-error-prone/src/main/java/com/uber/autodispose/errorprone/UseAutoDispose.java

```diff
@@ -92,8 +94,9 @@ public UseAutoDispose() {
   public UseAutoDispose(ErrorProneFlags flags) {
     Optional<ImmutableSet<String>> inputClasses =
         flags.getList("TypesWithScope").map(ImmutableSet::copyOf);
+    Optional<Boolean> exclusiveScope = flags.getBoolean("IsScopeExclusive");
 
-    ImmutableSet<String> classesWithLifecycle = inputClasses.orElse(DEFAULT_CLASSES_WITH_LIFECYCLE);
+    ImmutableSet<String> classesWithLifecycle = getClassesWithLifecycle(inputClasses, exclusiveScope);
```

Comment:
```suggestion
    ImmutableSet<String> classesWithLifecycle = getClassesWithScope(inputClasses, exclusiveScope);
```

Reply:
We can rename the field here too opportunistically

Reply:
5a99bfecf3cfe7c38cc97d77d8af0fd4ec28e308